quickstarts/docarray-hnswsearch/config.yml

```diff
@@ -0,0 +1,53 @@
+slug: docarray-hnswsearch
+description: |
+  ## Why should you monitor your usage of DocArray HnswSearch?
+  Monitor your vector searches on DocArray HnswSearch to get visibility on what you send to DocArray HnswSearch, responses retrieved from DocArray HnswSearch, latency, usage and errors.
+
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
```

Comment:
```suggestion
  Track the behavior of DocArray HnswSearch. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
```